lib/hammer_cli_foreman/ssh.rb

```diff
@@ -12,6 +12,7 @@ class Command < HammerCLIForeman::Command
 
     command_name 'SSH to hosts'
     option %w(-c --command), 'COMMAND', _('Command to execute'), :attribute_name => :command, :required => true
+    option %w(-n --concurrent), 'CONCURRENCY', _('Number of concurrent SSH sessions'), :attribute_name => :concurrent
```

Comment:
I guess if it's like before I'm OK with it - and see this PR even more valuable as it's making a limit possible. :dart: